### DISCUSSION TITLE
作者是放弃这个项目的更新了吗？

### DISCUSSION THREAD

**Discussion by im-shu** [Mzying2001/CefFlashBrowser#18] [Q&A] [answered]
接近半年没有更新了，是有什么变故吗？

**Comment by Mzying2001** [comment, marked as answer]
我目前是大三学生，也不是计算机相关专业的，平时没有多少时间搞这个，有时间的话我也许会更新一下

**Reply by im-shu** [reply]
目前希望能支持mac，或者支持windwos（arm版原生运行），支持64位，加入unity3d支持。谢谢
